src/client/components/Editor.tsx

```diff
@@ -15,6 +15,8 @@ type EditorProps = {
     settingsPaneIsOpen: boolean;
     onCodeChange: (code: string) => void;
     code: string;
+    breakpoints: number[];
+    addBreakpoint: (b: number) => void;
```

Comment:
I think it might be better to have a function that can be used for _any_ breakpoint change, since we also need to be able to handle removing breakpoints. Maybe something like:
```
onBreakpointChange: (breakpoints: number[]) => void;
```